### PR TITLE
feat: SRM - Scan type TAROT-2669

### PR DESCRIPTION
**Pull request by nicklem**
Documents the "Scan type" finding categorization in the context of SRM

### :eyes: Live preview
https://tarot-2669-doc-update-documentation--docs-codacy.netlify.app/organizations/managing-security-and-risk/#scan-types

### :construction: To do
- [x] Confirm scope with squad
- [x] Integrate with any missing info
- [x] Audit screenshots

### CODE REVIEW

File: docs/organizations/managing-security-and-risk.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ To access the findings page, access the [overview page](#dashboard) and click th
 
 ![Security and risk management findings page](images/security-risk-management-findings.png)
 
-When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, or **Security category** drop-downs above the list.
+When viewing the findings, you can update the filtering criteria by clicking the **Severity**, **Status**, **Repository**, **Security category**, or **Scan type** drop-downs above the list.
 
-To find out more about a finding in the list, click its **Details** column to navigate to the finding details on the source platform.
+To find out more about a finding in the list, you can navigate to the finding details on the source platform by clicking its **Details** column. This column also offers a quick overview of each finding, including its title, the source platform, the security category, and related information, such as the repository name, Jira issue key, or penetration testing report URL.
```
Comment (nicklem) [inline docs/organizations/managing-security-and-risk.md:96]:
```suggestion
To find out more about a finding in the list, you can navigate to the finding details on the source platform by clicking its **Details** column. This column also offers a quick overview of each finding, including its title, the source platform, the security category, and related information such as the repository name, Jira issue key, or penetration testing report URL.
```

File: docs/organizations/managing-security-and-risk.md [hunk 2]
```diff
@@ -445,3 +445,18 @@ Each Codacy issue reported by Security and risk management belongs to one of the
...
+
+The following table lists the available scan types and their descriptions:
+
+| Scan type                                   | Description                                                                                                               |
+|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| **Open-source dependencies**                | External libraries and packages with vulnerabilities or outdated versions.                                                |
+| **Exposed secrets**                         | Sensitive information, such as passwords or API keys, inadvertently included in the code.                                 |
+| **Infrastructure as code misconfiguration** | Configuration issues within infrastructure-as-code (IaC) files that could pose risks.                                     |
+| **Penetration testing**                     | Results from [penetration testing](#opening-and-closing-pen-testing-items) to find security vulnerabilities in your code. |
```
Comment (nicklem) [inline docs/organizations/managing-security-and-risk.md:460]:
```suggestion
| **Penetration testing**                     | Results from [penetration testing](#opening-and-closing-pen-testing-items) to find security vulnerabilities in your running code. |
```

File: docs/organizations/managing-security-and-risk.md [hunk 3]
```diff
@@ -445,3 +445,17 @@ Each Codacy issue reported by Security and risk management belongs to one of the
 | **Visibility**                     | Logging should always be included for security events to better allow attack detection and help defend against vulnerabilities.                                                                                  |
 | **XSS**                            | Cross-Site Scripting (XSS) attacks inject malicious client-side scripts into trusted websites that are visited by the end users.                                                                                 |
 | **Other**                          | Other language-specific security issues.                                                                                                                                                                         |
+
+## Scan types
+
+Each finding detected in an organization's repository is also categorized by **Scan type**, indicating the specific source or method used to detect the finding. This information helps you understand the origin of the finding and the context in which the underlying issue was detected.
+
+The following table lists the available scan types and their descriptions:
```
Comment (nicklem) [inline docs/organizations/managing-security-and-risk.md:453]: @codacy/tarot I'm limiting the list to what we have as confirmed for now. I [removed the upcoming ones](https://github.com/codacy/docs/pull/2128/commits/6ad8b0edee86d820942d509920c4015242e65d4e) for now; open to including some/all of them, on condition that we're *really* sure they're upcoming. Wouldn't want to set the wrong expectations, especially in docs.

File: docs/organizations/managing-security-and-risk.md [hunk 4]
```diff
@@ -445,3 +445,17 @@ Each Codacy issue reported by Security and risk management belongs to one of the
 | **Visibility**                     | Logging should always be included for security events to better allow attack detection and help defend against vulnerabilities.                                                                                  |
 | **XSS**                            | Cross-Site Scripting (XSS) attacks inject malicious client-side scripts into trusted websites that are visited by the end users.                                                                                 |
 | **Other**                          | Other language-specific security issues.                                                                                                                                                                         |
+
+## Scan types
```
Comment (claudiacarpinteiro) [inline docs/organizations/managing-security-and-risk.md:449]: I'm not sure if we are being consistent across the UI regarding the order of the different filters/dimensions (severity, status, security category, scan type)...
If we are, consider ordering the corresponding sub-sections following the same order, and moving [Languages checked for security issues](https://github.com/codacy/docs/pull/2128/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R220) to the end (as we don't have a filter for it).

**Review by claudiacarpinteiro — COMMENTED** [review]
Looks good, @nicklem! 👍 
I left some comments for your evaluation.

File: docs/organizations/managing-security-and-risk.md (hunk 4 above)
Comment (nicklem) [reply docs/organizations/managing-security-and-risk.md:449]: Good point

File: docs/organizations/managing-security-and-risk.md [hunk 5]
```diff
@@ -161,7 +161,23 @@ Codacy closes a finding when it detects that the associated Jira issue is marked
 !!! note
     Penetration testing is available upon request and is provided by a third-party partner. See [how to request penetration testing for your organization](https://go.codacy.com/pen-testing-product).
 
-Codacy opens a new finding when penetration testing results are published by our partner.
+Codacy opens a new finding when penetration testing results are published by our partner.<!-- TODO TAROT-2669: clarify when these are closed -->
+
+## Finding severities and deadlines {: id="item-severities-and-deadlines"}
```
Comment (nicklem) [inline docs/organizations/managing-security-and-risk.md:166]: Note: Feel free to ignore this section when reviewing. I moved it to ensure consistency with the UI.

File: docs/organizations/managing-security-and-risk.md [hunk 6]
```diff
@@ -201,21 +217,58 @@ The following table describes how finding statuses map to deadlines:
     </tbody>
 </table>
 
-## Finding severities and deadlines {: id="item-severities-and-deadlines"}
+## Supported security categories
```
Comment (nicklem) [inline docs/organizations/managing-security-and-risk.md:220]: Note: Feel free to ignore this section when reviewing. I moved it to ensure consistency with the UI.

File: docs/organizations/managing-security-and-risk.md [hunk 7]
```diff
@@ -161,7 +161,7 @@ Codacy closes a finding when it detects that the associated Jira issue is marked
 !!! note
     Penetration testing is available upon request and is provided by a third-party partner. See [how to request penetration testing for your organization](https://go.codacy.com/pen-testing-product).
 
-Codacy opens a new finding when penetration testing results are published by our partner.
+Codacy opens a new finding when penetration testing results are published by our partner.<!-- TODO TAROT-2669: clarify when these are closed -->
```
[the remark this answers is not in the document]
Comment (mrfyda) [reply docs/organizations/managing-security-and-risk.md:164]: still no updates.
we can say they're not closed automatically, as they'd be closed upon new pentesting scan or manually when requested.

**Review by mrfyda — APPROVED** [review]

**Pull request merged**